src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -1207,5 +1207,15 @@ CHIP_ERROR ExtractPubkeyFromX509Cert(const ByteSpan & certificate, Crypto::P256P
     return CHIP_ERROR_NOT_IMPLEMENTED;
 }
 
+CHIP_ERROR ExtractAKIDFromX509Cert(const ByteSpan & certificate, MutableByteSpan & akid)
+{
+    return CHIP_ERROR_NOT_IMPLEMENTED;
```

Comment:
Do we not need this implemented at some point?  This should at least be a TODO with a tracking issue filed.